### DISCUSSION TITLE
How can I update my existing Ghost3.X app already deployed on Heroku?

### DISCUSSION THREAD

**Discussion by sridesigns** [thelovekesh/ghost-v4-on-heroku#3] [Q&A] [answered]
Hi, I have already deployed 3.X version of Ghost on Heroku and I want to update to the latest version? Can someone assist me on how to do that?

**Comment by thelovekesh** [comment, marked as answer]
You have provided insufficient info. Provide the code repo or source from where you deployed version 3.

**Reply by sridesigns** [reply]
Apologies. Since I was yet to any create content on Ghost,  I removed the old deployed version and have installed your version on Heroku. The deployment works fine. 
But in future, how can I update this deployed version? (Sorry if this is a very basic question.. A programming newbie here 🙈)

**Reply by alan345** [reply]
Also interested in the answer!

**Reply by thelovekesh** [reply]
Once you have deployed the Ghost V4 from this repo it means that necessary `config vars` are set up on your Heroku dyno. To regularly update things what you can do is download this repo and run `npm update` to update all the packages and `npm update ghost`  for updating Ghost to the latest version. After updating the `packages` you can deploy the repo directly to your Heroku dyno.

**Reply by alan345** [reply]
Thanks for your answer, But there is still something I still font get. If I deploy the repo directly to my heroku dyno, it will erase my database, and my images uploaded?

**Reply by thelovekesh** [reply]
This is a discussion section about the project not about how particular things work. If you want help regarding deployment, changing themes, update ghost on Heroku, updating dependencies then create a `new topic` [here](https://ask.codebulbs.com). A detailed answer will be provided. Click [here](https://codebulbs.com) to create a topic.
